electron/src/index.ts

```diff
@@ -217,14 +217,21 @@ ipcMain.on('initiateUpdate', () => {
   }
 })
 
+const userNumber = process.env.ACORN_TEST_USER_2 ? "2" : ""
 const MIGRATION_FILE_NAME_PREFIX = `data-migration-for-integrity-version-`
+const dataPath = app.isPackaged ? app.getPath('userData') : path.join(__dirname, `../../user${userNumber}-data/`)
+const migrationFile = path.join(
+  dataPath,
+  `${MIGRATION_FILE_NAME_PREFIX}${INTEGRITY_VERSION_NUMBER}`
+)
+const prevIntegrityVersion = INTEGRITY_VERSION_NUMBER - 1
+const prevVersionMigrationFile = path.join(
+  dataPath,
+  `${MIGRATION_FILE_NAME_PREFIX}${prevIntegrityVersion}`
+)
 
```

Comment:
for best practice by convention, we could make all these global consts named using CAPITAL_UNDERSCORE casing. When reading functions, it helps to be able to quickly distinguish between local and global variables.